napalm/nxos/nxos.py

```diff
@@ -199,14 +199,17 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
         elif self.transport == 'http':
             self.port = optional_args.get('port', 80)
 
+        self.ssl_verify = optional_args.get('verify', False)
```

Comment:
You didn't change the second half of this i.e. optional_args.get is still looking for verify.